recipes/openff-toolkit/meta.yaml

```diff
@@ -10,16 +10,14 @@ source:
   sha256: 38eeab18f5e2c0eeeb06d96ef93397070319a7e55fc9ea61d806f45c43f82cdb
 
 build:
-  noarch: python
```

Comment:
I think this should stay.

Reply:
At some point, I got it in my head that using a build script meant this line had to be dropped. That seems to be wrong? I was able to build locally with your suggested changes